dashboard/test/controllers/openai_chat_controller_test.rb

```diff
@@ -7,24 +7,30 @@ class OpenaiChatControllerTest < ActionController::TestCase
     OpenaiChatHelper.stubs(:get_chat_completion_response_message).returns({status: 200, json: {}})
   end
 
-  # User without ai_chat_access is unable to access the chat completion endpoint
+  # User without ai tutor access is unable to access the chat completion endpoint
   test_user_gets_response_for :chat_completion,
-  user: :levelbuilder,
+  user: :student,
   method: :post,
   params: {messages: [{role: "user", content: "Say this is a test!"}]},
-  response: :forbidden
-
-  # With ai_chat_access, a post request without a messages param returns a bad request
-  test_user_gets_response_for :chat_completion,
-  user: :ai_chat_access,
-  method: :post,
-  params: {},
   response: :bad_request
 
-  # With ai_chat_access, a post request with a messages param returns a success
+  # User with ai tutor access disabled is unable to access the chat completion endpoint
+
+  # User with ai tutor access from permissions, post request with a messages param returns a success
   test_user_gets_response_for :chat_completion,
-  user: :ai_chat_access,
+  user: :ai_tutor_access,
   method: :post,
   params: {messages: [{role: "user", content: "Say this is a test!"}]},
   response: :success
+
+  # User with ai tutor access from experiment and section enablement, post request with a messages param returns a success
+
+  # A post request without a messages param returns a bad request
+  test_user_gets_response_for :chat_completion,
+  user: :ai_tutor_access,
+  method: :post,
+  params: {},
+  response: :bad_request
+
+  # A post request with a messages param returns a success
```

Comment:
- [ ] update section factory to include aiTutorEnabled 
- [ ] finish these tests